src/helper/ssg/index.ts

```diff
@@ -101,14 +101,15 @@ export const fetchRoutesContent = async <
   const baseURL = 'http://localhost'
 
   for (const route of inspectRoutes(app)) {
-    if (route.isMiddleware || isDynamicRoute(route.path)) continue
+    if (route.isMiddleware) continue
 
     // GET Route Info
     const thisRouteBaseURL = new URL(route.path, baseURL).toString()
     const forGetInfoURLRequest = new Request(thisRouteBaseURL) as AddedSSGDataRequest
     await app.fetch(forGetInfoURLRequest)
 
     if (!forGetInfoURLRequest.ssgParams) {
```

Comment:
Was this the flag to set the middleware? I guess I was thinking too hard.

Reply:
@watany-dev yessss